services/commands-service.ts

```diff
@@ -104,6 +105,11 @@ export class CommandsService implements ICommandsService {
 
 				this.$errors.fail("Unable to execute command '%s'. Use '$ %s %s --help' for help.", beautifiedName, this.$staticConfig.CLIENT_NAME.toLowerCase(), beautifiedName);
 				return false;
+			} else {
```

Comment:
You can simplify the code a bit writing ```else if``` (without nesting). Considering that everything in the if returns, the else is also not strictly needed, but I'd rather leave it so that future refactoring does not break easily.